plugins/inventory/iocage.py

```diff
@@ -357,6 +373,21 @@ def get_inventory(self, path):
 
                 results['_meta']['hostvars'][hostname]['iocage_hooks'] = iocage_hooks
 
+        # Optionally, get the jails names from the properties notes.
+        # Requires the notes format "t1=v1 t2=v2 ..."
+        if inventory_hostname_tag:
+            if not get_properties:
+                raise AnsibleError('Jail properties are needed. Enable get_properties')
+            update = {}
+            for hostname, host_vars in results['_meta']['hostvars'].items():
+                tags = dict([tag.split('=') for tag in host_vars['iocage_properties']['notes'].split()])
+                if inventory_hostname_tag in tags:
+                    update[hostname] = tags[inventory_hostname_tag]
+                elif inventory_hostname_required:
+                    raise AnsibleError(f'Mandatory tag {inventory_hostname_tag} is missing in the properties notes.')
```

Comment:
I'd add `!r` here to ensure that the value is quoted:
```suggestion
                    raise AnsibleError(f'Mandatory tag {inventory_hostname_tag!r} is missing in the properties notes.')
```